avocado_vt/plugins/vt_init.py

```diff
@@ -0,0 +1,180 @@
+
+            help_msg = 'Monitor type (human or qmp)'
+            settings.register_option(section, '',
+                                     help_msg=help_msg,
+                                     default='')
+
+            help_msg = 'Number of virtual cpus to use (1 or 2)'
+            settings.register_option(section, 'smp',
+                                     help_msg=help_msg,
+                                     default='2')
+
+            help_msg = 'Image format type to use (any valid qemu format)'
+            settings.register_option(section, 'image_type',
+                                     help_msg=help_msg,
+                                     default=SUPPORTED_IMAGE_TYPES[0])
+
+            help_msg = 'Guest network card model (any valid qemu card)'
+            settings.register_option(section, 'nic_model',
+                                     help_msg=help_msg,
+                                     default=SUPPORTED_NIC_MODELS[0])
+
+            help_msg = ('Guest disk bus for main image. One of ide, scsi, '
+                        'virtio_blk, virtio_scsi, lsi_scsi, ahci, usb2 '
+                        'or xenblk. Note: Older qemu versions and/or '
+                        'operating systems (such as WinXP) might not support '
+                        'virtio_scsi. Please use virtio_blk or ide instead.')
+            settings.register_option(section, 'disk_bus',
+                                     help_msg=help_msg,
+                                     default=SUPPORTED_DISK_BUSES[0])
+
+            help_msg = 'Enable qemu sandboxing (on/off)'
+            settings.register_option(section, 'sandbox',
+                                     help_msg=help_msg,
+                                     default='on')
+
+            help_msg = ('Prevent qemu from loading sysconfdir/qemu.conf '
+                        'and sysconfdir/target-ARCH.conf at startup (yes/no)')
+            settings.register_option(section, 'defconfig',
+                                     help_msg=help_msg,
+                                     default='yes')
+
+            help_msg = ('Use MALLOC_PERTURB_ env variable set to 1 to help '
+                        'catch memory allocation problems on qemu (yes/no)')
+            settings.register_option(section, 'malloc_perturb',
+                                     help_msg=help_msg,
+                                     default='yes')
+
+            # [vt.libvirt] section
+            help_msg = ('Test connect URI for libvirt (qemu:///system, '
+                        'lxc:///)')
+            settings.register_option('vt.libvirt', 'connect_uri',
+                                     help_msg=help_msg,
+                                     default='qemu:///session')
+
+            # [vt.debug] section
+            help_msg = ('Do not clean up tmp files or VM processes at the end '
+                        'of a virt-test execution')
+            settings.register_option('vt.debug', '',
```

Comment:
the option name used to be `no_cleanup` but now is empty string, seems that forgot to update?
```
$ avocado config | grep 'vt\.debug'
    vt.debug.                                False
```

Reply:
Yes, just noticed this too

Reply:
There is also another missing string I see:
```
    vt.qemu.vhost                           off
    vt.qemu.                                
    vt.qemu.smp                             2
```

Reply:
I also notice that the keys are not sorted and I have some `vt` options followed by `plugins` options then again by `vt` options. This is suboptimal since a reader would want to find all keys in a given namespace at one location in the output of `avocado config`.

Reply:
> There is also another missing string I see:
> 
> ```
>     vt.qemu.vhost                           off
>     vt.qemu.                                
>     vt.qemu.smp                             2
> ```

That was the `monitor` key, fixed for v2.  Thanks for spotting it too.

Reply:
> I also notice that the keys are not sorted and I have some `vt` options followed by `plugins` options then again by `vt` options. This is suboptimal since a reader would want to find all keys in a given section at one location in the output of `avocado config`.

Yeah, I followed the original order of the argparser based code in order to try to keep my sanity :).  I'll try to reorder them now.

Reply:
> > I also notice that the keys are not sorted and I have some `vt` options followed by `plugins` options then again by `vt` options. This is suboptimal since a reader would want to find all keys in a given section at one location in the output of `avocado config`.
> 
> Yeah, I followed the original order of the argparser based code in order to try to keep my sanity :). I'll try to reorder them now.

Actually, I think I misunderstood it.  You meant on `avocado config`.  Yeah, this is something to be fixed on the Avocado side indeed: https://github.com/avocado-framework/avocado/issues/4086

Reply:
Ah great, will be keeping an eye on avocado-framework/avocado#4086.